src/main/plugin/iso19139.ca.HNAP/templates/hnaplevel1-english.xml

```diff
@@ -119,9 +119,8 @@
             </gmd:CI_Contact>
          </gmd:contactInfo>
          <gmd:role>
-            <gmd:CI_RoleCode codeList="http://nap.geogratis.gc.ca/metadata/register/napMetadataRegister.xml#IC_90"
+            <gmd:CI_RoleCode codeList="https://schemas.metadata.geo.ca/register/napMetadataRegister.xml#IC_90"
                              codeListValue="RI_414">pointOfContact; contact</gmd:CI_RoleCode>
-         </gmd:role>
```

Comment:
Is there a reason that you removed the ending element?

Reply:
I do not believe this was intentional; it may of been a mistake in backport conflict resolution.

Reply:
Actually Ian it is one line down, I expect this is a whitespace change.